packages/pwa/app/pages/cart/index.jsx

```diff
@@ -35,7 +35,7 @@ import {API_ERROR_MESSAGE} from '../../constants'
 import {REMOVE_CART_ITEM_CONFIRMATION_DIALOG_CONFIG} from './partials/cart-secondary-button-group'
 
 // Utilities
-import debounce from 'lodash.debounce'
+import debounce from 'lodash/debounce'
```

Comment:
I don't see lodash being installed, and the `lodash.debounce` being removed. We need to consider whether installing the whole package of lodash makes sense because lodash bundle size is round [70kb ](https://bundlephobia.com/package/lodash@4.17.21) while `lodash.debounce` is just [2kb](https://bundlephobia.com/package/lodash.debounce@4.0.8)

Reply:
Using `lodash/debounce` amounts to pretty much the same thing as `lodash.debounce` - it only loads the files necessary for `debounce`. That being said, `debounce` is pretty trivial to implement if we want to move away from `lodash` entirely.

Reply:
Agree with @wjhsf, we might just want to implement it ourselves. I looked around we can use [this](https://www.educative.io/edpresso/how-to-use-the-debounce-function-in-javascript) a reference
```
function debounce(func, wait, immediate) {
  var timeout;

  return function executedFunction() {
    var context = this;
    var args = arguments;
	    
    var later = function() {
      timeout = null;
      if (!immediate) func.apply(context, args);
    };

    var callNow = immediate && !timeout;
	
    clearTimeout(timeout);

    timeout = setTimeout(later, wait);
	
    if (callNow) func.apply(context, args);
  };
};
```

Reply:
We still use the `lodash` dependency in the SDK:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/91ffe0e3f096327d0a1648fe125e56a6bc391b90/packages/pwa-kit-react-sdk/package.json#L70
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/88fb0eabcc4fd95ebc15a391dc7660c4ab24fbba/packages/pwa-kit-react-sdk/scripts/file-utils.js#L10

Thus, we still need to file a 3PP request. The 3PP for `lodash` is approved: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qAH000000041qYAA/view

Since removing `lodash` dependency from the monorepo will require some work including making sure we don't introduce a breaking change, I propose to do that on a separate ticket.

The change to use `lodash/debounce` instead of `lodash.debounce` on V1 was merely done to be consistent with the change done in V2, removing the dependency `lodash.debounce` from the PWA. Making more visible that we're no longer using the `lodash.debounce` dependency.
https://github.com/SalesforceCommerceCloud/pwa-kit/pull/488/files#diff-1f1fc5de05465ee4931aab2618da9f4a0fc5ef41459ba425936b1c5e46f7ac2dL39